packages/transaction-controller/src/TransactionController.ts

```diff
@@ -237,7 +237,7 @@ export class TransactionController extends BaseControllerV1<
 
   // TODO: Replace `any` with type
```

Comment:
Should we take care of this TODO now?

Reply:
I did initially try, but it caused additional type issues so thought best to limit the scope since it's a bug fix.